src/lib.rs

```diff
@@ -290,6 +290,23 @@
 //! # fn main() {}
 //! ```
 //!
+//! ## Environment Variable Fallback
+//!
+//! It is possible to specify an environment variable fallback option for arguments, so that their value is taken
+//! from the specified environment variables if not given through the command-line:
```

Comment:
I think it should be "variable" in the singular here.

Reply:
👍 fixed